composer.json

```diff
@@ -35,5 +35,11 @@
       "email": "github-sr@hotmail.com",
       "role": "Maintainer"
     }
-  ]
+  ],
+  "require-dev": {
+    "macopedia/phpstan-magento1": "^1.0"
+  },
+  "extra": {
+    "magento-root-dir": "."
```

Comment:
not sure if this is a good idea, or if it causes any unexpected problems

Reply:
No idea, this was added by composer plugin after ‚composer install’.
If its not needed I can remove it.